openmc/material.py

```diff
@@ -908,22 +899,51 @@ def get_nuclide_atom_densities(self):
 
         return nuclides
 
-    def get_nuclide_activity(self):
-        """Return activity in [Bq] for each nuclide in the material
+    def get_activity(self, normalization: str = 'volume', by_nuclide: bool = False):
+        """Returns the activity of the material or for each nuclide in the
+        material in units of [Bq], [Bq/g] or [Bq/cc].
 
         .. versionadded:: 0.13.1
 
+        Parameters
+        ----------
+        normalization : {'total', 'mass', 'volume'}
+            Specifies the type of activity to return, 'total' will return the
+            material activity in [Bq], 'mass' returns the materials specific
+            activity in [Bq/g] and 'volume' returns the activity in [Bq/cc].
```

Comment:
Just to throw out an alternative -- how about having this argument by `units` that accepts values "Bq", "Bq/g", and "Bq/cc"? Might be a little more obvious to a user. This would also lend itself to allowing other units (e.g., Ci) if we wanted.

Reply:
Also, in other places in the API, we use "cm3" instead of "cc", so it might be better to have that last option be "Bq/cm3" for the sake of consistency.

Reply:
I can get these done, thanks for the improvements